en/integrations/plugin-packs/procedures/cloud-aws-ec2.md

```diff
@@ -3,102 +3,262 @@ id: cloud-aws-ec2
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisistes
+
+### AWS Configuration
+
+Configure an access/secret key combo these privileges:
+
+| AWS Privilege                  | Description                                                     |
+| :----------------------------- | :-------------------------------------------------------------- |
+| ec2:DescribeInstances          | Display EC2 instances & ASG details                             |
+| ec2:DescribeSpotFleetRequests  | Display EC2 Spot Fleet Requests details                         |
+| cloudwatch:getMetricStatistics | Get metrics from the AWS/EC2 namespace on Cloudwatch            |
+
+### Plugin dependencies
+
+To interact with Amazon APIs, you can use either use awscli binary or paws, a perl AWS SDK (recommended). You must install it on every poller that will monitor AWS resources. 
+
+**Warning** At the moment it is not possible to use perl-Paws if you are using a proxy to reach AWS Cloudwatch APIs. 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--perl-Paws-installation-->
+
+```bash
+yum install perl-Paws
+```
+
+<!--aws-cli-installation-->
+
+```bash
+yum install awscli
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Setup 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Install the Centreon Plugin package on every poller expected to monitor Amazon EC2 ressources:
+
+```bash
 yum install centreon-plugin-Cloud-Aws-Ec2-Api
 ```
 
-To use it, you can either install 'awscli' (AWS Command Line Interface) or
-'paws' (Perl AWS SDK).
+2. Install the monitoring templates from the Centreon Plugin-Pack on the "Configuration > Plugin packs > Manager" page
 
-### Install awscli
 
-On CentOS, install with following commands:
+<!--Offline IMP License-->
 
-``` shell
-yum install awscli
+1. Install the Centreon Plugin package on every poller expected to monitor Amazon RDS ressources:
+
+```bash
+yum install centreon-plugin-Cloud-Aws-Ec2-Api
 ```
 
-## Centreon Configuration
+2. Install the Centreon Plugin-Pack RPM:
 
-### Create a host using the appropriate template
+```bash
+yum install centreon-pack-cloud-aws-ec2.noarch
+```
 
-#### For an AutoScalingGroup
+3. Install the monitoring templates from the Centreon Plugin-Pack on the "Configuration > Plugin packs > Manager" page
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-| Field            | Value                    |
-| :--------------- | :----------------------- |
-| Name             | *Name of the host*       |
-| Alias            | *Description*            |
-| IP Address / DNS | *Can be localhost*       |
-| Monitored from   | *Poller used to monitor* |
-| Templates        | Cloud-Aws-Ec2-Asg-custom |
+## Configuration
 
-The following host macros should be set as shown:
+- Log into Centreon and add a new host through "Configuration > Hosts".
+- Apply the relevant Host Template. They all begins with "Cloud-Aws-Ec2*".  
```

Comment:
"host" but "Host Template" ? We should discuss and state a standard in the objects naming and stick to it

Reply:
Indeed Host must be capitalized